### PR TITLE
Extend Import/Export documentation with Remote Storage

### PR DESCRIPTION
**Pull request by adifucan**
## Purpose of this pull request

This pull request (PR) extends Import/Export documentation with remote storage functionality.

## Magento release version

- [x] 2.4.2

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source
- [x] Magento Commerce

## Affected documentation pages:
- src/system/data-export.md
- src/system/data-import.md
- src/system/data-import-history.md
- src/system/data-import-product-images.md
- src/system/data-schedule-export.md
- src/system/data-schedule-import.md
- src/system/data-transfer-downloadable-products.md
- src/system/data-transfer-examples.md

## Additional information

Here is a PR with this functionality to 2.4-develop that is going to be merged: https://github.com/magento-commerce/magento2ce/pull/6344

### CODE REVIEW

File: src/system/data-import-product-images.md [hunk 1]
```diff
@@ -36,17 +36,17 @@ Before you try to import a large amount of images, review the steps in the impor
 
 1. Follow the instructions to [import]({% link system/data-import.md %}) the data.
 
-1. After selecting the file to import, enter the relative path following **Images File Directory**:
+1. After selecting the file to import, enter the relative path following **Images File Directory** `var/import/images`:
 
     ```
-    var/import/images
+    e.g. product_images
```
Comment (leslietilling) [inline src/system/data-import-product-images.md:42]:
```suggestion
var/import/images/product_images
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Some minor edits and linting fixes

**Review by hguthrie — CHANGES_REQUESTED** [review]
I cleaned up a few things.
I think that there is a lot of repetition about remote storage enablement, but it looks like there is a good reason. @leslietilling should there be a link to devdocs about enabling Remote storage? Or should there be a section somewhere in Merchant docs about Enabling remote storage...?

File: src/system/data-export.md [hunk 2]
```diff
@@ -49,3 +49,5 @@ Data export is an asynchronous operation, which executes in the background so th
 1. Upon completion of the task, look for the file in the download location for your web browser.
 
     You can save or open the exported CSV file as a spreadsheet, then edit the data and import it back into your store.
+    {:.bs-callout-info}
+    By default, all exported files are located in the `<Magento root directory>/var/export` folder. If remote storage like Amazon S3 is enabled, all exported files are located in the `<Remote storage root directory>/import_export/export` folder.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/system/data-export.md:53]: @hguthrie  We should link to devdocs for enabling remote storage. I don't know of anything to do that via the Admin.

**Comment by leslietilling** [comment]
> I cleaned up a few things.
> I think that there is a lot of repetition about remote storage enablement, but it looks like there is a good reason. @leslietilling should there be a link to devdocs about enabling Remote storage? Or should there be a section somewhere in Merchant docs about Enabling remote storage...?

@hguthrie We should link to devdocs for enabling remote storage. I don't know of anything to do that via the Admin.

**Comment by hguthrie** [comment]
Sorry for the delay. Holidays and much more going on. I will wrap up this edit now that the Config Guide updates are in final editorial review. The config guide Remote storage section refers to the Schedule Imports/Exports topic in the User guide. I will see where it might help to link back to config...

**Review by hguthrie — APPROVED** [review]
I finished reviewing. I do think that the repeated note is a candidate for include file, so I would like to merge this into a new branch to make that structural change before creating a new PR to 2.4.2-develop for release.

**Pull request merged**